autopilot/interface.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/lightningnetwork/lnd/lnwire"
 )
 
+// DefaultConfTarget is the default confirmation target for autopilot channels.
+// TODO(halseth): possibly make dynamic, going aggressive->lax as more channels
+// are opened.
+const DefaultConfTarget = 3
```

Comment:
Once https://github.com/lightningnetwork/lnd/pull/3140 is in, we could even make this higher and users can bump manually if desired.